idea/gearexe/frontend/src/shared/config/consts.ts

```diff
@@ -0,0 +1,29 @@
+export const PROJECT_ID = import.meta.env
+  .VITE_WALLET_CONNECT_PROJECT_ID as string;
+export const GEAR_API_NODE = import.meta.env.VITE_GEAR_EXE_API_NODE as string;
+export const ETH_CHAIN_ID = 17000; // (0x4268) Holesky
+export const ETH_NODE_ADDRESS = import.meta.env.VITE_ETH_NODE_ADDRESS;
+
+export const DIGIT_RECOGNITION_CONTRACT_ADDRESS = import.meta.env
+  .VITE_DIGIT_RECOGNITION_CONTRACT_ADDRESS as `0x${string}`;
+
+enum AnimationTimeout {
```

Comment:
I would suggest not to utilize enums, `as const` objects should be sufficient.

[This approach is opinionated](https://www.youtube.com/watch?v=jjMbPt_H3RQ), but [TS docs](https://www.typescriptlang.org/docs/handbook/enums.html#objects-vs-enums) suggests we might not need them as well.